app/assets/javascripts/area_served.js

```diff
@@ -0,0 +1,45 @@
+
+
+
+function calculate_client_share_total(){
+    total = 0;
+    console.log("*********mark 1 total is: " +total);
```

Comment:
This should be `console.debug` or removed entirely.

Reply:
Yes.  Oversight.  Will be corrected in next push.